Learning_test_automation/Projekt#1/TeachersGradebook/TeachersRoom/GradeFile.cs

```diff
@@ -0,0 +1,78 @@
+﻿using SchoolData;
+
+namespace Infrastructure
+{
+    public class GradeFile
+    {
+        FileWizard fileWizard = new FileWizard();
+
+        public void GradeFileCreation(string createText)
+        {
+            string uniqName = "Grade";
+            fileWizard.SaveDataToFile(uniqName, createText);
+        }
+
+        public void AddGradeParametersFromFileToList()
+        {
+            string fileName = @"C:\Users\lemk\Desktop\TeachersGradebook\Grade.txt";
+
+            if (File.Exists(fileName))
+            {
+                using (StreamReader sr = new StreamReader(fileName))
+                {
+                    if (sr.Peek() >= 0)
+                    {
+                        string fileContent = sr.ReadToEnd();
+
+                        string[] grades = fileContent.Split('\n');
+
+                        SchoolSubjects schoolSubject = default;
+                        double gradeValue    = default;
+                        int studentId     = default;
+                        int gradeID = default;
+
+                        foreach (var lineWithGradeDate in grades)
+                        {
+                            string[] fields = lineWithGradeDate.Split(';');
+
+                            foreach (string field in fields)
+                            {
+                                if (field.Trim().StartsWith("School subject:"))
```

Comment:
Parametry dla metod `StartsWith` powinny byc trzymane jako `const string`, bo kazda z nich jest wywolywana 2 razy i po prostu powinny byc stalymi.

Reply:
powiem szczerze, że zrobiłem to ale tak średnio mi się podoba więc dla studenta pozostawię jak było bo może zmienisz zdanie jak to zobaczysz :D